attributes/default.rb

```diff
@@ -105,7 +105,7 @@
 #Log level of REST API
 default['hopsworks']['hopsworks_rest_log_level'] = "PROD"
 
-default['hopsworks']['mysql_connector_url']         = "#{node['download_url']}/mysql-connector-java-5.1.29-bin.jar"
+default['hopsworks']['mysql_connector_url']         = "#{node['download_url']}/mysql-connector-java-8.0.21-bin.jar"
```

Comment:
I guess, since they changed the classpath, that the connector for 5.1.29 and 8.0.21 can co-exist without issues - but can you double check it please?

Reply:
Otherwise we have to remove the previous version during an upgrade.

Reply:
does not seem to create any problmes